fre/tests/test_fre_cli.py

```diff
@@ -24,11 +24,11 @@ def test_cli_fre_option_dne():
 
 def test_fre_version():
     ''' module import flavor of below cli test '''
-    assert '2024.01' == fre.version
+    assert '2025.01' == fre.version
```

Comment:
I guess this is ok.. we'll have to update this test every release.

Reply:
Yeah I wasn't sure about this one, definitely wanted to check

Reply:
updating the version explicitly is the point. if the version is not updated everywhere, the pipeline SHOULD prevent us from publishing.

Reply:
like- if you are changing either this `assert` statement, or `fre.version`, and you forgot to change it elsewhere, the pipeline should absolutely bonk a dev on the head.